src/demo/renderer.js

```diff
@@ -27,6 +27,10 @@ export const initRenderer = () => {
 };
 
 const canvasCache = new CanvasCache();
+const intermediateCanvas = document.createElement('canvas');
```

Comment:
Should we maybe put this inside `initRenderer`?  I encountered an issue developing last night where these module-level variables were being instantiated before we'd really initialized anything else in our app because `$(document).ready` hadn't yet fired, and it was a bit confusing.  It feels like we should hold off on module-specific business logic until our app has gotten up and running...

Reply:
Yeah I was torn about that but that makes sense. I moved the `intermediateCanvas`, `intermediateCtx, and `cacheCanvas` instantiation into `initRenderer`.

Reply:
Thanks.  @madelynkasula, this discussion might be of interest in general too.

Reply:
ah, good points. let's talk about this in standup today